playbook/spec/pb_kits/playbook/kits/date_picker_spec.rb

```diff
@@ -26,6 +26,7 @@
   it { is_expected.to define_prop(:picker_id).of_type(Playbook::Props::String).that_is_required }
   it { is_expected.to define_prop(:placeholder).of_type(Playbook::Props::String) }
   it { is_expected.to define_prop(:plugins).of_type(Playbook::Props::Boolean).with_default(false) }
+  it { is_expected.to define_prop(:selection_type).of_type(Playbook::Props::Enum).with_default("month") }
```

Comment:
This has to be updated as well.

```suggestion
  it { is_expected.to define_prop(:selection_type).of_type(Playbook::Props::Enum).with_default("none") }
```